roles/jd-client/src/lib/mod.rs

```diff
@@ -415,3 +415,41 @@ impl PoolChangerTrigger {
         }
     }
 }
+
+#[cfg(test)]
```

Comment:
Can you please move this test to https://github.com/stratum-mining/stratum/pull/1360/commits/de393492d4cf3dbb8ff8efc8d9d216a338629c62